portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DeleteSegments.testcase

```diff
@@ -588,9 +580,9 @@ definition {
 		task ("Add a jobTitle field") {
 			ACSegments.goToSidebarAttributes(criterion = "Individual Attributes");
```

Comment:
The "task" still says "Add a jobTitle field" even though it is using email

Reply:
Ohh nice, thanks for noticing that!
I was reviewing here and I'm seeing I forgot to add some minor things. I will fix it soon.